Sources/News.php

```diff
@@ -333,12 +338,15 @@ function buildXmlFeed($xml_format, $xml_data, $feed_meta, $subaction)
 	$forceCdataKeys = array();
 	$nsKeys = array();
 
+	// Maybe someone needs to insert a DOCTYPE declaration?
+	$dtd = '';
```

Comment:
Same here, better use a descriptive name.

Reply:
This is descriptive. :) DTD is the common abbreviation for Document Type Definition. It's even [used in the XML spec itself](https://www.w3.org/TR/xml/#sec-prolog-dtd).

Reply:
Yes, inside their context its descriptive, inside SMF's context it is not. Chances are someone looking at the code will not have knowledge of the XML spec to the point of easily identify its meaning.

Reply:
Fair enough.